secboot/secboot_dummy.go

```diff
@@ -35,3 +36,12 @@ func SealKeys(keys []SealKeyRequest, params *SealKeysParams) error {
 func ResealKeys(params *ResealKeysParams) error {
 	return fmt.Errorf("build without secboot support")
 }
+
+func WriteKeyData(path string, encryptedPayload, auxKey []byte, rawhandle *json.RawMessage) error {
+	return fmt.Errorf("build without secboot support")
+}
+
+func MarshalKeys(key []byte, auxKey []byte) []byte {
+	panic("build without secboot support")
+	return nil
+}
```

Comment:
Now that SealKeysWithFDESetupHook exits I don't think we will have to export these, or so I hope.